rdr_service/tools/tool_libs/genomic_utils.py

```diff
@@ -2211,11 +2211,15 @@ def run(self):
         for set_member in set_members:
             if self.args.results_only:
                 set_member.blockResults = 0
+                set_member.blockResultsReason = f"Formerly blocked due to '{set_member.blockResultsReason}'"
             elif self.args.research_only:
                 set_member.blockResearch = 0
+                set_member.blockResearchReason = f"Formerly blocked due to '{set_member.blockResearchReason}'"
             else:
                 set_member.blockResults = 0
                 set_member.blockResearch = 0
+                set_member.blockResultsReason = f"Formerly blocked due to '{set_member.blockResultsReason}'"
+                set_member.blockResearchReason = f"Formerly blocked due to '{set_member.blockResearchReason}'"
```

Comment:
Should we just be checking the type of "blocked" it is @ execution and then update the reason? Or if you are passing the type in the args can we have a check to see if it was actually blocked so we won't have discrepancies in the data such as "Formerly blocked due to None".